core/src/test/java/com/linecorp/armeria/server/streaming/JsonLinesTest.java

```diff
@@ -0,0 +1,141 @@
+package com.linecorp.armeria.server.streaming;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Stream;
+
+import org.junit.ClassRule;
+import org.junit.Test;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.google.common.collect.ImmutableList;
+import com.google.common.util.concurrent.MoreExecutors;
+
+import com.linecorp.armeria.client.WebClient;
+import com.linecorp.armeria.common.AggregatedHttpResponse;
+import com.linecorp.armeria.common.HttpData;
+import com.linecorp.armeria.common.HttpObject;
+import com.linecorp.armeria.common.HttpResponse;
+import com.linecorp.armeria.common.HttpStatus;
+import com.linecorp.armeria.common.MediaType;
+import com.linecorp.armeria.common.ResponseHeaders;
+import com.linecorp.armeria.server.ServerBuilder;
+import com.linecorp.armeria.testing.junit4.server.ServerRule;
+
+import io.netty.handler.codec.http.HttpHeaderNames;
+import reactor.core.publisher.Flux;
+import reactor.test.StepVerifier;
+
+public class JsonLinesTest {
+    private static class Pojo {
+        @JsonProperty("name")
+        private final String name;
+        @JsonProperty("age")
+        private final int age;
+        @JsonProperty("cars")
+        private List<String> cars;
+        public Pojo(String n, int a, List<String> c) {
+            name = n;
+            age = a;
+            cars = c;
+        }
+
+    }
+    private static final ObjectMapper mapper = new ObjectMapper();
+
+    @ClassRule
+    public static ServerRule rule = new ServerRule() {
```

Comment:
We prefer to use JUnit5 for new test suites.

Reply:
switched it to junit5